modules/azurerm/Cosmos-Database-Account/variables.tf

```diff
@@ -148,3 +148,15 @@ variable "ip_range_filter" {
   description = "A set of IP addresses or IP address ranges in CIDR form to be included as the allowed list of client IPs"
   type        = string
 }
+
+variable "network_acl_bypass_for_azure_services" {
+  default     = true
```

Comment:
Can we set default value as false

Reply:
fixed with [d3b6cbb](https://github.com/wso2/azure-terraform-modules/pull/114/commits/d3b6cbb74c00ecd6afeb379d98d943e084f66776)